core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -457,7 +457,6 @@ object TestUtils extends Logging {
     val producerProps = props.getOrElse(new Properties)
     producerProps.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList)
     producerProps.put(ProducerConfig.ACKS_CONFIG, acks.toString)
-    producerProps.put(ProducerConfig.MAX_BLOCK_MS_CONFIG, maxBlockMs.toString)
```

Comment:
This change causes the parameter of `createNewProducer` to be ignored affecting a bunch of other tests though. It seems like the easiest thing is to add a couple of protected methods in `IntegrationTestHarness`: `createProducer` and `createConsumer`. You can then override that in `EndToEndAuthorizationTest`.